### DISCUSSION TITLE
Inherit Zoom for browse pages

### DISCUSSION THREAD

**Discussion by l0oo** [sumatrapdfreader/sumatrapdf#2751] [Ideas]
Hello!
Is it possible to add such feature like 'inherit zoom' for browse pdf pages?
I'd like to use just page down/page up and have inherited zoom for all pages...
Thanks for this awesome app! :)

PS
In Acrobat, it is possible to force browsing pages (with inherit zoom) as bookmarks (plugin needed + this feature... have no own separate shortcut - not possible 'go to next/previous bookmark' - its panel need to be open all the time).

**Comment by l0oo** [comment]
So... Is that possible? :/

**Comment by GitHubRulesOK** [comment]
In theory the bookmarks should "ENFORCE" A zoom so a page can have multiple bookmarks for different objects (AFAIK that worked! in SumatraPDF so untested recently) The main problem is that the zoom needs to be very small in area to not be affected by page boundary requirements, so often seem to have no effect.

The page up page down should be using current zoom (whatever that is at present) if it is in a variable mode such as width the width will override the zoom.

So unclear as to what inherited zoom means in such cases.
